v0/destinations/vero/data/VeroIdentifyConfig.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "sourceKeys": "userId",
+    "destKey": "id",
+    "sourceFromGenericMap": true
+  },
```

Comment:
Rather than doing this in transform.js, we're mapping it in the order you've mentioned.
```
{
    "sourceKeys": ["userId", "context.traits.email", "traits.email", "anonymousId"],
    "destKey": "id"
}
```

Reply:
This mapping is not correct either `userId` or `anonymousId` would be present in the payload, otherwise the payload wont reach here.
The email should be mapped to `email` field which is mentioned in the api docs

Reply:
Updated